### DISCUSSION TITLE
How to activate Copilot for Github Organization?

### DISCUSSION THREAD

**Discussion by remixtedi** [community/community#40209] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

One of the features for GitHub Organizations says that there is an EXCLUSIVE ADD-ON of Copilot for a free package of Organization, But there are no instructions about how can I activate it from my device, can you help me solve this? I am not using Copilot on my private account.

Github Organizations page: https://github.com/organizations/plan

**Comment by davecheney** [comment, marked as answer]
@remixtedi I am sorry, the text on the website is misleading. Free organisations are not entitled to copilot for free. I am working with the relevant parties to correct this misleading information. Thank you.
